sites/www_mozilla_org.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "www_mozilla_org",
     "location": "ec2-us-west-1:Chrome",
-    "connection": "3G",
+    "connection": "Native",
```

Comment:
I'd like to stick the the "default" being 3G for our sites, since this is probably the more useful stress test for front-end devs. Changing this here would also make the graph confusing when it jumps from 3g to native, so I'd suggest leaving this as-is. If you want to add a native speed test, I'd suggest using a new json file for it.